CreeDictionary/API/apps.py

```diff
@@ -62,6 +65,58 @@ def read_morpheme_rankings():
             Wordform.MORPHEME_RANKINGS[morpheme] = float(freq)
 
 
+def initialize_affix_search():
+    """
+    build dictionaries to facilitate affix search
+    """
+    # how many letters from the start/end do we check, larger length means more memory used, longer initialization time, but faster search
+    length = 5
```

Comment:
I'm confused.... the length here is 5... but later the length is 4?

---

In addition, could you please extract this into a constant in `settings.py`? Some name like `CREE_DICTIONARY_AFFIX_SEARCH_LENGTH`.

Reply:
what's more, this length and the later length is not neccessarily the same. e.g. We can check the beginning 3 letters and the ending 3 letters but we might choose to do the affix search for words longer than, say, 10 letters (which won't be the case but just to explain my point).

Reply:
Oh geez. Even more reason to give them both constant names! ;)